app/lib/common/widgets/visibility/visibility_chip.dart

```diff
@@ -51,24 +50,17 @@ class VisibilityChip extends ConsumerWidget {
     }
     return Chip(
       padding: const EdgeInsets.symmetric(horizontal: 2),
-      backgroundColor: Theme.of(context).colorScheme.neutral,
       visualDensity: VisualDensity.compact,
-      shape: RoundedRectangleBorder(
-        borderRadius: const BorderRadius.all(Radius.circular(5)),
-        side: BorderSide(
-          color: Theme.of(context).colorScheme.neutral3,
-        ),
+      shape: const RoundedRectangleBorder(
+        borderRadius: BorderRadius.all(Radius.circular(5)),
```

Comment:
If I am not mistaken, we opted for a color here as the chip is bright white and that's way to intense and overstates it's importance in the UI (and thus is bad UX). Just removing the muted colors with complete disregard for its purpose and reason seems ... careless...